File: 프로그래머스/lv0/120818. 옷가게 할인 받기/옷가게 할인 받기.js
function solution(price) {
    let answer = 0;
    if (price >= 500000) {
        answer = price * (1 - 0.20);
    } else if (price >= 300000) {
        answer = price * (1 - 0.10);
    } else if (price >= 100000) {
        answer = price * (1 - 0.05);
    } else {
        answer = price;
    }
    answer = Math.floor(answer);
    console.log(answer);
    return answer;
}

File: 프로그래머스/lv0/120818. 옷가게 할인 받기/옷가게 할인 받기.java
class Solution {
    public int solution(int price) {
        int answer = 0;
        if (price >= 500000) {
            answer = (int)(price * (1 - 0.20));
        } else if (price >= 300000) {
            answer = (int)(price * (1 - 0.10));
        } else if (price >= 100000) {
            answer = (int)(price * (1 - 0.05));
        } else {
            answer = (int)(price);
        }
        System.out.println(answer);
        return answer;
    }
}